File: Person.py
class Person:
    def __init__(self,name,last_name,birth_date,curp):
        self.name = name
        self.last_name = last_name
        self.birth_date = birth_date
        self.curp = curp
    
    def get_name(self):
        return self.name
    def set_name(self,name):
        self.name = name
    def set_last_name(self,last_name):
        self.last_name = last_name
    def set_birth_date(self,birth_date):
        self.birth_date = birth_date
    def set_curp(self,curp):
        self.curp = curp

File: Person.java
public class Person {
    private String name;
    private String lastname;
    private String birthDate;
    private String curp;

    public Person(String name, String lastname, String birthDate, String curp) {
        this.name = name;
        this.lastname = lastname;
        this.birthDate = birthDate;
        this.curp = curp;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public void setBirthDate(String birthDate) {
        this.birthDate = birthDate;
    }

    public void setCurp(String curp) {
        this.curp = curp;
    }

    public String showInfo(){
        return String.format("Nombre completo: %s %s, Fecha de nacimiento: %s, CURP: %s", name, lastname, birthDate, curp);
    }
}
